### ISSUE TITLE
Strings 6394–6395 (+ various changed strings)

### ISSUE THREAD

**Issue by AaronVanGeffen** [OpenRCT2/Localisation#1979]
Updated strings on PR OpenRCT2/OpenRCT2#13186.

Rephrased strings:

```diff
-STR_2753    :Mowed grass
+STR_2753    :Mow grass
-STR_2767    :Freeze weather
+STR_2767    :Disable weather changes
-STR_5725    :{BLACK}Force weather:
+STR_5725    :{BLACK}Change weather to:
-STR_5726    :{SMALLFONT}{BLACK}Sets the current weather in park
+STR_5726    :{SMALLFONT}{BLACK}Sets the current weather in the park
-STR_5871    :Plants don't age
+STR_5871    :Disable plants withering
```

Capitalisation changes:

```diff
-STR_2765    :Large Tram
+STR_2765    :Large tram
-STR_2769    :Open Park
+STR_2769    :Open park
-STR_2770    :Close Park
+STR_2770    :Close park
-STR_6331    :Create Ducks
+STR_6331    :Create ducks
-STR_6332    :Remove Ducks
+STR_6332    :Remove ducks
```

New strings:
```
STR_6394    :Objective
STR_6395    :Maintenance
```

Updated in:
- [ ] ar-EG
- [ ] ca-ES
- [ ] cs-CZ
- [ ] da-DK
- [ ] de-DE
- [x] en-US
- [ ] eo-OO
- [ ] es-ES
- [ ] fr-FR
- [ ] hu-HU
- [ ] it-IT
- [ ] ja-JP
- [ ] ko-KR
- [ ] nb-NO
- [ ] nl-NL
- [ ] pl-PL
- [ ] pt-BR
- [ ] sv-SE
- [ ] tr-TR
- [ ] zh-CN
- [ ] zh-TW

Notification when new strings arrive:
@Omaranwa @J0anJosep @octaroot @LPSGizmo @danidoedel @Wuzzy2 @tellovishous @JasperGray @Wirlie @JoelTroch @anon569 @LucaRed @AaronVanGeffen @telk5093 @Goddesen @Gymnasiast @renansimoes @Tupaschoal @Nubbie @izhangfei @daihakken @marcinkunert

**Comment by frenchiveruti** [comment]
What's "Large Tram" or what's the context, because in spanish someone before me translated STR_2765 as "Visitors group"

**Comment by tupaschoal** [comment]
Clicking it spawns a looong queue of peeps

**Comment by frenchiveruti** [comment]
So it's more of a metaphor than an actual tram, thanks.

**Comment by Gymnasiast** [comment]
Yup. The wording was directly taken from 8 cars per trainer (I think the Son of Beast trainer also used it).

**Comment by frenchiveruti** [comment]
Is it okay if I change it to a tooltip that's actually useful like "Bring Visitors"?

**Comment by frenchiveruti** [comment]
I'm a moron.

**Comment by LucaRed** [comment]
> Is it okay if I change it to a tooltip that's less poetic like "Bring Visitors"?

@frenchiveruti I had the same problem in Italian, but I think I've found a good translation. I'm writing it here for future reference (and as a possible suggestion for other translators).

I've translated "Large tram" as "Trenino di visitatori", where "trenino" is the Italian expression for "bailar la conga" (as an added bonus, "trenino" also means "small train").

**Comment by Gymnasiast** [comment]
@frenchiveruti @LucaRed Please do not feel compelled to make a literal translation. A translation is about _what_ is being said, not _how_. Just make sure your translation make senses in Spanish/Italian etc. and don't worry if you deviate from the original English.

**Comment by frenchiveruti** [comment]
Sure thing, maybe in the future I'll change it to "Conga de visitantes" just for fun.  
For now, the translation is the meaning of the button.  
It should probably have a tooltip but doesn't really affect anything.

**Comment by galfisk** [comment]
Partially applied to sv-SE.

Two terms still have their old translations.
STR_2767, should be changed to: Avaktivera väderförändringar
STR_5725, should be changed to: Ändra väder till:
